Makefile

```diff
@@ -104,6 +109,32 @@ docker-build: test ## Build docker image with the manager.
 docker-push: ## Push docker image with the manager.
 	docker push ${IMG}
 
+##@ Build Dependencies
+
+## Location to install dependencies to
+LOCALBIN ?= $(shell pwd)/bin
+$(LOCALBIN):
+	mkdir -p $(LOCALBIN)
+
+
+.PHONY: operator-sdk
```

Comment:
This block of script code is part of the standard Makefile generated by operator-sdk